core/spoofax.compiler/src/main/java/mb/spoofax/compiler/adapter/AdapterProjectCompiler.java

```diff
@@ -163,6 +165,10 @@ public None compile(ExecContext context, Input input) throws IOException {
         addTaskDef(allTaskDefs, input.checkAggregatorTaskDef(), input.baseCheckAggregatorTaskDef());
         addTaskDef(allTaskDefs, input.checkDeaggregatorTaskDef(), input.baseCheckDeaggregatorTaskDef());
 
+        if (input.strategoRuntime().isPresent() && input.constraintAnalyzer().isPresent()) {
```

Comment:
Can we support testing Stratego strategies without having a constraint analyzer? Not all languages will use the constraint analyzer but do want to test Stratego strategies.